workflows/epitopeprediction.nf

```diff
@@ -263,6 +263,8 @@ workflow EPITOPEPREDICTION {
     EXTERNAL_TOOLS_IMPORT(
         ch_nonfree_paths
     )
+    // TODO: Is there a way to provide and "empty" path object if netmhc is not given?
+    ch_exported_tools = EXTERNAL_TOOLS_IMPORT.out.nonfree_tools.ifEmpty(".")
```

Comment:
What happens if you remove this line and just use `EXTERNAL_TOOLS_IMPORT.out.nonfree_tools` below ?

Reply:
It throws an error that a path needs to be specified in the peptide_prediction process if you don't use netmhc

Reply:
So there needs to some sort of a check wether the path has been emitted or not

Reply:
I think it is suggested to use `[]` here instead of `"."`.